src/common/indexer/elastic/elastic.indexer.service.ts

```diff
@@ -886,7 +884,7 @@ export class ElasticIndexerService implements IndexerInterface {
   async getAllFungibleTokens(): Promise<any[]> {
     const query = ElasticQuery.create()
       .withMustMatchCondition('type', TokenType.FungibleESDT)
-      .withFields(["name", "type", "currentOwner", "numDecimals", "properties", "timestamp"])
+      .withFields(["name", "type", "currentOwner", "numDecimals", "properties", "timestamp", 'ownersHistory'])
```

Comment:
use single or double quotes for string values in the entire line

Reply:
done